### DISCUSSION TITLE
Button groups in tables overflow weirdly

### DISCUSSION THREAD

**Discussion by domenkozar** [fomantic/Fomantic-UI#732] [Collections (form, grid, menu, message, table, ...)] [answered]
# Bug Report

## Steps to reproduce

1. https://jsfiddle.net/swxo02br/
2. make sure to make the window mobile small to see it overflow

## Expected result

I'd expect button group to be withing the cell and expand content-wise inside the table.

## Actual result

Overflows the table.

## Screenshot (when possible)

![2019-05-09-115645_954x242_escrotum](https://user-images.githubusercontent.com/126339/57428219-e34f9000-7216-11e9-9605-905b21e6926d.png)

## Version

2.7.4

**Comment by y0hami** [comment]
I would argue that this is invalid since your table has `unstackable` and you are also explicitly specifying the width of the table columns therefore your table is technically working correctly. I would also argue that you shouldn't use a `unstackable` table on mobile because its never going to work in the first place.

If you disagree feel free to explain why.

**Comment by domenkozar** [comment]
That's a valid point, but even if stackable is used it overflows before it goes to stack mode.

**Comment by lubber-de** [comment, marked as answer]
@domenkozar Wrap both buttons into one `ui group basic vertical buttons` (instead of two single `group buttons`)
The second button (means the column width you defined) is still too small, but the content does wrap now instead of overflow:

See https://jsfiddle.net/fko624wq/

![image](https://user-images.githubusercontent.com/18379884/57464066-57704f00-727c-11e9-8529-824dc5b7ad57.png)

**Comment by domenkozar** [comment]
@lubber-de thanks that works perfectly :)
